project/build.scala

```diff
@@ -50,7 +50,8 @@ object Geoprocessing extends Build {
   )
 
   val resolutionRepos = Seq(
-    "Job Server Bintray" at "https://dl.bintray.com/spark-jobserver/maven"
+    Resolver.bintrayRepo("spark-jobserver", "spark-jobserver"),
```

Comment:
It certainly does not harm, but is the resolver for `spark-jobserver` necessary for this project?